server/webapp/WEB-INF/rails.new/app/assets/javascripts/pipeline_configs/views/run_if_conditions_widget.js.msx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define(['mithril', 'lodash', 'string-plus', '../helpers/form_helper'],
```

Comment:
Do we really need this as a widget?

It won't be used any where other than tasks widget. 
Even the model, should've been scoped inside the task model as it will not be constructed without the tasks part of the json.


The tasks_config_widget_spec now does not have a spec for run if making run if widget isolated which i think is wrong.